stringsAssignment.js

```diff
@@ -0,0 +1,85 @@
+// 1. console.log the first character of a string.
+let var1 = "String";
+console.log(var1[0]);
+
+// 2. console.log the length of a string.
+let var2 = "StringTwo";
+console.log(var2.length);
+
+// 3. console.log the last character of any string.
+let var3 = "String3";
+console.log(var3[var3.length - 1]);
+
+// 4. Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
+//
+// ```js
+// 'jimmy'
+// // => 'jimmY'
+// ```
+let var4 = "StringFour";
```

Comment:
Good use of function here but function is outputting "R" not "StringFouR". How can we fix this?